src/targets/c/libcurl/fixtures/multipart-data.c

```diff
@@ -8,6 +8,6 @@ struct curl_slist *headers = NULL;
 headers = curl_slist_append(headers, "content-type: multipart/form-data; boundary=---011000010111000001101001");
 curl_easy_setopt(hnd, CURLOPT_HTTPHEADER, headers);
 
-curl_easy_setopt(hnd, CURLOPT_POSTFIELDS, "-----011000010111000001101001\r\nContent-Disposition: form-data; name=\"foo\"; filename=\"hello.txt\"\r\nContent-Type: text/plain\r\n\r\nHello World\r\n-----011000010111000001101001\r\nContent-Disposition: form-data; name=\"bar\"\r\n\r\nBonjour le monde\r\n-----011000010111000001101001--\r\n");
```

Comment:
Fun side effect of the quicky non-native `FormData` work we had set up was that sometimes the `filename` in a HAR would get basenamed. This isn't happening anymore anywhere, and I think that's okay. The HARs we ingest for this library in the browser will always have a basenamed filename thanks to browser security not giving us full paths

Reply:
I checked the [golang source](https://github.com/golang/go/blob/a03f1aeb557e0e4547c7dfdc1bd64d1f6f3a56ec/src/mime/multipart/multipart.go#L396-L400) to confirm that the trailing newlines are not required, and it agrees